utoipa-swagger-ui/build.rs

```diff
@@ -102,7 +103,7 @@ impl SwaggerZip {
             let mut file = self.by_index(index)?;
             let filepath = file
                 .enclosed_name()
-                .ok_or(ZipError::InvalidArchive("invalid path file"))?;
+                .ok_or(ZipError::InvalidArchive(Cow::Borrowed("invalid path file")))?;
```

Comment:
This works on version 2.4.2 or 2.5 of zip and if your dependencies are pinned to something below 2.5, you may want compatibility with both versions of `zip`.
```suggestion
                .ok_or(ZipError::InvalidArchive("invalid path file".into()))?;
```

Reply:
thanks @RossRogers